src/services/bitcoin-rpc.service.ts

```
@@ -24,11 +25,20 @@ export class BitcoinRpcService implements OnModuleInit {
 
     async onModuleInit() {
         const url = this.configService.get('BITCOIN_RPC_URL');
-        const user = this.configService.get('BITCOIN_RPC_USER');
-        const pass = this.configService.get('BITCOIN_RPC_PASSWORD');
+        let user = this.configService.get('BITCOIN_RPC_USER');
+        let pass = this.configService.get('BITCOIN_RPC_PASSWORD');
         const port = parseInt(this.configService.get('BITCOIN_RPC_PORT'));
         const timeout = parseInt(this.configService.get('BITCOIN_RPC_TIMEOUT'));
 
+        const cookiefile = this.configService.get('BITCOIN_RPC_COOKIEFILE')
```

ESLint

[error] Replace `········const·cookiefile·=·this.configService.get('BITCOIN_RPC_COOKIEFILE')` with `····const·cookiefile·=·this.configService.get('BITCOIN_RPC_COOKIEFILE');`

(prettier/prettier)